src/tools/android/java/com/google/devtools/build/android/desugar/LambdaDesugaring.java

```diff
@@ -44,14 +41,25 @@
 import org.objectweb.asm.tree.InsnNode;
 import org.objectweb.asm.tree.MethodNode;
 import org.objectweb.asm.tree.TypeInsnNode;
+import org.objectweb.asm.tree.analysis.Analyzer;
+import org.objectweb.asm.tree.analysis.BasicInterpreter;
+import org.objectweb.asm.tree.analysis.BasicValue;
+import org.objectweb.asm.util.Printer;
+import org.objectweb.asm.util.Textifier;
+import org.objectweb.asm.util.TraceMethodVisitor;
+
+import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
I think the square formatter got to you. I think these need to go at the top. Does bazel code use google-java-format?

Reply:
Ayup. Fixed.